<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiConfigClass</class>
 <widget class="QDialog" name="UiConfigClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>546</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UiConfig</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;</string>
       </property>
       <property name="text">
        <string>滚动速度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="speedEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;
font: 36pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>200</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;</string>
       </property>
       <property name="text">
        <string>滚动间隔</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="intervalEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;
font: 36pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>100</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;</string>
       </property>
       <property name="text">
        <string>图片宽度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="widthEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;
font: 36pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>300</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;</string>
       </property>
       <property name="text">
        <string>图片高度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="heightEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;
font: 36pt &quot;黑体&quot;;</string>
       </property>
       <property name="text">
        <string>500</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="confirmBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;</string>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 36pt &quot;AniMe Matrix - MB_EN&quot;;</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
